azure/notebooks/azure_mnmg.ipynb

```diff
@@ -0,0 +1,674 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Multi-node multi-GPU example on Azure using dask-cloudprovider\n",
+    "\n",
+    "[Dask Cloud Provider](https://cloudprovider.dask.org/en/latest/) is a native cloud intergration for dask. It helps manage Dask clusters on different cloud platforms. In this notebook, we will look at how we can use the package to set-up a Azure cluster and run a multi-node, multi-GPU example with [RAPIDS](https://rapids.ai/). RAPIDS provides a suite of libraries to accelerate data science pipelines on the GPU entirely. This can be scaled to multiple nodes using Dask as we will see through this notebook. \n",
+    "\n",
+    "For the purposes of this demo, we will use the NYC Taxi Dataset and run on a part of it."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Before running the notebook, run the following commands in the terminal to setup Azure CLI\n",
+    "```\n",
+    "pip install azure-cli\n",
+    "az login\n",
+    "```\n",
+    "\n",
+    "The list of packages needed for this notebook is listed in the cell below - uncomment and run the cell to set it up."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# !pip install \"dask-cloudprovider[azure]\"\n",
+    "# !pip install \"dask-cloudprovider[azure]\" --upgrade\n",
+    "# !pip install --upgrade azure-mgmt-network azure-mgmt-compute\n",
+    "# !pip install gcsfs\n",
+    "# !pip install dask_xgboost\n",
+    "# !pip install azureml"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from dask_cuda import LocalCUDACluster\n",
```

Comment:
Also I don't think the `LocalCUDACluster` is used here

Reply:
@zronaghi I thought it'd be better to give instructions for docker setup in a README along with the notebook. It might be confusing to have docker + conda instructions. What do you think?

@jacobtomlinson Removing it now.

Reply:
Sounds good, we can add the docker/setup instructions in the README